samples/CommunityToolkit.Maui.Sample/Pages/Views/LazyView/CustomLazyView.cs

```diff
@@ -0,0 +1,21 @@
+﻿using CommunityToolkit.Maui.Markup;
+
+namespace CommunityToolkit.Maui.Sample.Pages.Views.LazyView;
+
+public class CustomLazyView<TView> : Maui.Views.LazyView where TView : View, new()
+{
+	public override async ValueTask LoadViewAsync()
```

Comment:
Should this have a

```suggestion
   /// <inheritdoc/>
	public override async ValueTask LoadViewAsync()
```

??

Reply:
The same app doesn't require XML Comments